scripts/pipelines/gcloud/pipeline_generator.sh

```diff
@@ -110,7 +113,7 @@ function addTriggers {
             echo -e "${green}Previous pipeline defined. Adding trigger inside: ${localDirectory}/${pipelinePath}/${previousPipelineyaml}.${white}."
             sed -e "s/# mark to insert trigger/- name: gcr.io\/cloud-builders\/gsutil\n  entrypoint: bash\n  args:\n  - -c\n  - |\n    if [[ "\$BRANCH_NAME" =~ $branchTrigger ]] || exit 0; then\n      token=\$(gcloud auth print-access-token)\n      curl -H \"Content-Type: application\/json; charset=utf-8\" -X POST --data '{\"substitutions\":{\"_BRANCH_NAME\":\"'\${BRANCH_NAME}'\"},\"commitSha\":\"'\${COMMIT_SHA}'\"}\' \"https:\/\/cloudbuild.googleapis.com\/v1\/projects\/\${PROJECT_ID}\/triggers\/$pipelineName:run?access_token=\${token}&alt=json\"\n      fi/g" $localDirectory/$pipelinePath/$previousPipelineyaml -i
             ;;
-        "package-pipeline.yml")
+        "*package-pipeline.yml")
```

Comment:
"*package-pipeline.yml" was not working for @mcerverc in yesterday's demo. He had to use:
```suggestion
        "package-pipeline.yml" || "flutter-package-pipeline.yml")
```
Still pending to discuss with @albertdb if package should use a separate pipeline definition YAML for Flutter

Reply:
Let me try it again but I tried it and it worked